trino-connector/src/main/java/com/datastrato/gravitino/trino/connector/catalog/CatalogConnectorManager.java

```diff
@@ -360,6 +368,12 @@ public void alterCatalog(
   }
 
   public void addMetalake(String metalake) {
+    if (config.simplifyCatalogNames()) {
+      if (!usedMetalakes.isEmpty())
```

Comment:
What is the size of `usedMetalakes` is 1 and what's the meaning of `Multiple metalakes`?

Reply:
A `usedMetalakes` value of 1 indicates that the user have configured one Gravitino catalog using a metalake. 
When the user add a new Gravitino catalog and using another metalake,  it''s running on  multiple catalogs mode